tools/github/before_install.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -ex
+
+export PIP_DEFAULT_TIMEOUT=60
+
+# This URL is for any extra wheels that are not available on pypi.  As of 14
+# Jan 2017, the major packages such as numpy and matplotlib are up for all
+# platforms.  The URL points to a Rackspace CDN belonging to the scikit-learn
+# team.  Please contact Olivier Grisel or Matthew Brett if you need
+# permissions for this folder.
+EXTRA_WHEELS="https://5cf40426d9f06eb7461d-6fe47d9331aba7cd62fc36c7196769e4.ssl.cf2.rackcdn.com"
+WHEELHOUSE="--find-links=$EXTRA_WHEELS"
+
```

Comment:
We don't have this Rackspace storage anymore, but it doesn't look like we really need it for this script anyway since these two environment variables don't appear to be used below?

I had setup Travis and Anaconda to upload wheels to anaconda over at scikit-image-wheels recently. To use those we would configure `EXTRA_WHEELS="https://pypi.anaconda.org/multibuild-wheels-staging/label/main/simple/"`.

Reply:
My understanding is that this extra wheel URL would previously have been used to find prerelease wheels for testing? It looks like now prereleases from PyPI are used instead via `pip --pre`.